installers/rpm-code/codes/flash.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+flash_main() {
+    codes_dependencies hypre
+    codes_download_proprietary "flash/FLASH-4.6.2.tar.gz" "FLASH4.6.2"
+    local flash_src_root=$PWD
+    patch_makefile
+    for n in CapLaser RTFlame
+    do
+        setup_$n $n
+        make_and_install $n
+    done
+}
+
+make_and_install() {
+    cd $1
+    codes_make
+    install -m 755 flash4 "${codes_dir[bin]}"/flash4-$1
```

Comment:
Quote `"$1"`. Document the `flash4-$1` with a POSIT so we know what has to match.

Reply:
https://github.com/radiasoft/download/pull/95/commits/4b4294d531035f961f3421d9e7450d2b8d2c07b2